### DISCUSSION TITLE
Candidatura di: Andrea Guzzo

### DISCUSSION THREAD

**Discussion by JeyDi** [Il-Libro-Open-Source/book#26] [Candidature Ambassadors In Corso]
Questo sondaggio viene aperto per tutto il mese di Agosto per indicare la preferenza o l'opposizione sulla candidatura a ruolo di Ambassador per:

[Andrea Guzzo](https://github.com/JeyDi)

In caso di disaccordo, si protegge l'anonimato ma sarebbe gradita una spiegazione tramite commento a questo sondaggio con delle motivazioni.

Attualmente, vista la fase embrionale del progetto, non ci saranno limiti minimi di votazione e la semplice superiorità dei voti a favore sarà considerata sufficiente a procedere con la nomina.

**Comment by Cadienvan** [comment]
Vista la quantità di voti positivi ricevuti e la mancanza di negazioni plausibili, a breve riceverai l'invito al team Ambassadors dell'organizzazione "Il Libro Open Source".

Welcome! 🚀
